### PR TITLE
Fix(domain): add tooltip to DNS updating message

### PR DESCRIPTION
**Pull request by rfougier**

### CODE REVIEW

File: client/app/domain/general-informations/GENERAL_INFORMATIONS.html [hunk 1]
```diff
@@ -58,7 +58,8 @@ <h3 class="oui-tile__title" data-i18n-static="domain_dashboard_general_informati
                                                         data-ng-if="!ctrlTabGeneralInformations.dnsStatus.isOk">
                                                 </button>
                                                 <small class="ml-3"
-                                                       data-ng-if="ctrlTabGeneralInformations.dnsStatus.refreshAlert">
+                                                       data-ng-if="ctrlTabGeneralInformations.dnsStatus.refreshAlert"
+                                                       data-uib-tooltip="{{::i18n.domain_dashboard_tooltip_dns_updating}}">
```
Comment (antleblanc) [inline client/app/domain/general-informations/GENERAL_INFORMATIONS.html]: `data-uib-tooltip-append-to-body="true"` isn't required here?

File: client/app/domain/dns/lock/domain-dns-lock.controller.js [hunk 2]
```diff
@@ -19,7 +19,10 @@ angular.module("controllers").controller(
             return this.Domain
                 .updateNameServerType(this.$stateParams.productId, "hosted")
                 .then(() => this.Alerter.success(this.$scope.i18n.domain_tab_DNS_lock_success, this.$scope.alerts.main))
-                .catch((err) => this.Alerter.alertFromSWS(this.$scope.i18n.domain_tab_DNS_lock_error, err, this.$scope.alerts.main))
+                .catch((err) => {
+                    _.set(err, "type", err.type || "ERROR");
+                    this.Alerter.alertFromSWS(this.$scope.i18n.domain_tab_DNS_lock_error, err, this.$scope.alerts.main);
```
Comment (FredericEspiau) [inline client/app/domain/dns/lock/domain-dns-lock.controller.js]: We should have different methods for error messages, information messages, etc...

File: client/app/domain/general-informations/GENERAL_INFORMATIONS.html [hunk 3]
```diff
@@ -78,8 +79,8 @@ <h3 class="oui-tile__title" data-i18n-static="domain_dashboard_general_informati
                                                       <span class="fa fa-info-circle" aria-hidden="true"></span>
                                                   </span>
                                                 <div>
-                                                    <span data-ng-bind="tr('domain_dashboard_dns_management_anycast_' + ctrlTabGeneralInformations.domain.hasDnsAnycast)"></span>
-                                                    <span data-ng-if="ctrlTabGeneralInformations.domain.hasDnsAnycast === null">
+                                                    <span data-ng-bind="tr('domain_dashboard_dns_management_anycast_' + (ctrlTabGeneralInformations.domain.hasDnsAnycast == null ? 'null' : ctrlTabGeneralInformations.domain.hasDnsAnycast.toString()))"></span>
+                                                    <span data-ng-if="ctrlTabGeneralInformations.domain.hasDnsAnycast == null">
```
Comment (FredericEspiau) [inline client/app/domain/general-informations/GENERAL_INFORMATIONS.html]: Don't use `==`, use `angular.isBoolean` instead

File: client/app/domain/general-informations/GENERAL_INFORMATIONS.html (hunk 1 above)
Comment (rfougier) [reply client/app/domain/general-informations/GENERAL_INFORMATIONS.html]: Nope

**Pull request merged**